src/expressions.rst

```diff
@@ -1719,7 +1719,23 @@ The :t:`evaluation` of a :t:`shift left expression` proceeds as follows:
    The :t:`right operand` is evaluated.
 
 #. :dp:`fls_f0p70y92k14f`
-   ``core::ops::Shl::shl(left_operand, right_operand)`` is invoked.
+   If the type of the left :t:`[operand]` is any :t:`[unsigned integer type]`
+   and the type of the right :t:`[operand]` is any :t:`[integer type]`, then
+   the :t:`[shift left expression]` evaluates to the left logical shift of the
```

Comment:
This introduces the concept of a left logical shift which is not defined (likewise for arithmetic below)